test/index.js

```diff
@@ -1200,14 +1204,14 @@ describe('def', function() {
 
   it('supports record types', function() {
     eq(typeof $.RecordType, 'function');
-    eq($.RecordType.length, 1);
-    eq($.RecordType.toString(), 'RecordType :: StrMap Type -> Type');
+    eq($.RecordType.length, 2);
+    eq($.RecordType.toString(), 'RecordType :: Type -> StrMap Type -> Type');
 
     //  Point :: Type
-    var Point = $.RecordType({x: $.Number, y: $.Number});
+    var Point = $.RecordType($.Any, {x: $.Number, y: $.Number});
```

Comment:
Every record type defined in the test suite currently extends `$.Any`. It would be good to include your `Point3D` example here as well.